ansible/playbook_Test_Scripts/Test_Script

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+printf "Enter the URL:  "			#User Input section
+read gitURL
+printf "Retain the VM? [y/n]\n" 
+read vmChoice
+
+
+cd ~/documents/					
+mkdir Unix_PB_Test_Folder
```

Comment:
typically `mkdir -p` . also add `|| true` at the end (run `shellcheck` to see why).